compiler/quilt/tools/command.py

```diff
@@ -1026,7 +1026,7 @@ def _worker_thread():
                                 else:
                                     with lock:
                                         tqdm.write("Download failed for %s: %s" % (obj_hash, ex))
-                                break
+                                    break
```

Comment:
ROFL re "un-break" that's hilarious.

Reply:
Damn, that wasn't even intentional... Didn't realize it until you pointed it out